Tasks/SqlAzureDacpacDeployment/README.md

```diff
@@ -10,6 +10,8 @@ The task is used to deploy Azure SQL Database to an existing Azure SQL Server, a
 - Rollback on Failure - If true, then the upgrade is rolled back if errors are encountered.
 - Backup Database Before Changes - If true, a backup of the database is taken prior to applying the changes.
 
+_NOTE: Currently, the Azure SQL Database Deployment task works only for **RDFE endpoints**. Support for ARM endpoints would be enabled shortly._
```

Comment:
Is the statement on ARM required. Why can't we just stop with the statement to call out current support for RDFE.